src/07-quote/editor.scss

```diff
@@ -7,4 +7,84 @@ Text Domain: mojblocks
 
 **********************************************************************************************/
 
-@import 'style.scss';
\ No newline at end of file
+/** Variables */
+$mojblocks-color-teal: #336C83;
+$mojblocks-color-yellow: #FFDD00;
+$mojblocks-color-purple: #912B88;
+$mojblocks-color-lime: #B8EA23;
+$mojblocks-color-violet: #4C2C92;
+$mojblocks-color-mustard: #FFCE21;
+$mojblocks-color-dark-blue: #003078;
+$mojblocks-color-orange: #FF993C;
+$mojblocks-color-black: #0B0C0C;
+$mojblocks-color-dark-grey: #454A4C;
+$mojblocks-color-grey: #6F777B;
+$mojblocks-color-mid-grey: #B1B4B6;
+$mojblocks-color-light-grey: #F3F2F1;
+$mojblocks-color-white: #ffffff;
+
+.mojblocks-quote {
+    &__image {
+        background-color: $mojblocks-color-dark-grey;
+        min-height: 70px;
+        border-radius: 4px;
+
+        &__button {
+            border-radius: 50%;
+            width: 50px;
+            height: 50px;
+            border: 3px solid #ffffff;
+            padding: 6px;
+            box-shadow: none;
+            transition: all 0.3s;
+            position: absolute;
+            top: 10px;
+
+            > svg {
+                color: white;
+            }
+
+            &:hover,
+            &:focus {
+                box-shadow: none !important;
+            }
+
+            &-add {
+                left: 10px;
+            }
+
+            &-change {
+                background-color: rgba(0, 0, 0, 0.8);
+                left: 10px;
+
+                > svg {
+                    color: white;
+                }
+
+                &:hover > svg,
+                &:focus > svg  {
+                    color: $mojblocks-color-grey;
+                }
+            }
+
+            &-remove {
+                background-color: rgba(0, 0, 0, 0.8);
+                left: unset;
+                right: 10px;
```

Comment:
Minor thing, but one thing to do (maybe as part of tidying up, separately) is to make sure that we use rems and have a px fallback. Not a blocker to merging this in though, maybe this could be added to a 'tidying up' card once we've got MVP done :)

Reply:
Aw :) this is only rendered in the editor, not the frontend.